litmus-portal/frontend/src/views/Analytics/ApplicationDashboards/Steps/TuneTheQueries/EditPanelsWizard/index.tsx

```diff
@@ -221,7 +221,9 @@ const EditPanelsWizard: React.FC<EditPanelsWizardProps> = ({
       panel_id: '',
       panel_group_id: '',
       created_at: '',
-      panel_group_name: 'Untitled Group name',
```

Comment:
Change to English as it will be only working for custom metrics that too when user does not update this.